### PR TITLE
add a bio file for nicban2008

### PR DESCRIPTION
**Pull request by nicban2008**
This pull request adds a bio file for @nicban2008
As requested in #11

### CODE REVIEW

**Comment by nicban2008** [comment]
This is the best!

**Comment by nicban2008** [comment]
I'm commenting again

**Comment by nbessa** [comment]
Sorry - I don't have any changes to suggest :(

File: nicban2008.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Github Nicban2008
+- **Name:** Nicban2008
+- **Occupation:**Web Developer II
```
Comment (WubbaDuck) [inline nicban2008.md:3]: What is your main programming language as a web developer?

Comment (nicban2008) [inline nicban2008.md:3]: Front end mostly HTML,CSS, javascript,Jquery, a little php, also velocity

**Pull request merged**